bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/impl/SingleProbeStrategy.scala

```diff
@@ -44,13 +44,11 @@ case class SingleProbeStrategy(key: AgsStrategyKey, params: SingleProbeStrategyP
 
   override def candidates(ctx: ObsContext, mt: MagnitudeTable): Future[List[(GuideProbe, List[SiderealTarget])]] = {
     val empty = Future.successful(List((params.guideProbe: GuideProbe, List.empty[SiderealTarget])))
-    def filterOnMagnitude(q: CatalogQuery, t: SiderealTarget): Boolean = {
-      params.referenceMagnitude(t).exists(q.filterOnMagnitude(t, _))
-    }
+
     // We cannot let VoTableClient to filter targets as usual, instead we provide an empty magnitude constraint and filter locally
     catalogQueries(ctx, mt).strengthR(backend).headOption.map(Function.tupled(VoTableClient.catalog)).map(_.flatMap {
         case r if r.result.containsError => Future.failed(CatalogException(r.result.problems))
-        case r                           => Future.successful(List((params.guideProbe, r.result.targets.rows.filter(t => filterOnMagnitude(r.query, t)))))
+        case r                           => Future.successful(List((params.guideProbe, r.result.targets.rows)))
```

Comment:
VoTable does not support mag filtering so we get all targets on a given radius, then mag filtering is local.
The strategy decides based on the context (`catalogQueries(ctx, mt)`) what to filter out based on rules defined on the magnitude
I want to avoid the second filtering step so that the CatalogNavigator is agnostic to the context or observation, it just get query params, essentially, coordinates, radius and a set of filters. the second step will mean special-casing on the CatalogNavigator